radio/src/targets/common/arm/CMakeLists.txt

```diff
@@ -12,11 +12,6 @@ option(DEBUG_INTERRUPTS "Count interrupts" OFF)
 option(DEBUG_TASKS "Task switching statistics" OFF)
 option(DEBUG_TIMERS "Time critical parts of the code" OFF)
 
-if(TIMERS EQUAL 3)
```

Comment:
I prefer you leave the option as it is, just set 3 as default
I personnally use only 2